monitoring/README.md

```diff
@@ -55,3 +102,315 @@ git pull
 
 scripts/deploy.sh prod
 ```
+
+This job is also set to run nightly via a `cronjob` to auto-generate our list of node operators:
+
+```
+0 0 * * * cd ~/audius-protocol/monitoring && scripts/deploy.sh prod
+```
+
+## Grafana
+
+Grafana is the main UI for our metrics. Grafana primarily relies on Prometheus as its metrics backend.
+
+### Adding New Dashboards
+
+Official dashboard names should be prefixed with `Audius - ` while personal dashboards should be prefixed with your name, for example: `Joaquin - `.
+
+Try to keep the number of personal dashboards low to maintain navigability.
+
+Our dashboards use common set of Variables (Dashboard `Settings` -> `Variables`):
+
+* `env`: `label_values(audius_dn_flask_route_latency_seconds_count, environment)`
+* `host`: `label_values(audius_dn_flask_route_latency_seconds_count{environment=~"$env"}, host)`
+
+To simplify the process of setting up dashboards each time, we can navigate to the `Audius - Boilerplate` dashboard's `Settings` -> `Save As...` dialog to copy the boilerplate.
+
+Additionally, follow the official documentation for [best practices for creating dashboards](https://grafana.com/docs/grafana/latest/best-practices/best-practices-for-creating-dashboards/).
+
+### Adding New Panels
+
+Plenty of complexity can be added when writing `PromQL`, but most times the following example queries will work for a majority of panels.
+
+However, before writing out PromQL from scratch, it may be easier to find a panel similar to the new panel that will be created, then:
```

Comment:
It would be great if we could have links to good starter panels like a good histogram and a good gauge

Reply:
but probably should be in a notion doc vs public README